src/Components/Dashboard/SendNFT/sendNft.module.css

```diff
@@ -423,6 +423,7 @@
   border-top-right-radius: 12px;
   border-top-left-radius: 12px;
   pointer-events: none;
+  vertical-align: unset;
```

Comment:
@marco-calderon-primelab We're using scss in this project so if there is any changes in css then do it in .scss file and .css file will be generated automatically from the .scss file.

Reply:
Sure, I'll make the proper adjustment.